core/subsurface-qt/DiveObjectHelper.cpp

```diff
@@ -67,6 +67,16 @@ DiveObjectHelper::DiveObjectHelper(struct dive *d) :
 {
 }
 
+DiveObjectHelper::operator bool() const
```

Comment:
this comment is about the commit message for "Mobile: Generate DiveObjectHelpers on the fly" - note the typo "On the flip side, NOT the DiveListModel..."

Reply:
Fixed.